### PR TITLE
Fix tests on Windows

### PR DESCRIPTION
**Pull request by TrySound**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ function gulpEslint(options) {
 
 	function verify(str, filePath) {
 		var result = linter.executeOnText(str, filePath).results[0];
+		result.filePath = path.relative(process.cwd(), result.filePath);
```
Comment (shinnn) [inline index.js]: Better fixed in https://github.com/adametry/gulp-eslint/commit/21cf2ef475c598aa82d56ba496e62f9764f547ce. Absolute paths are more useful for end users.

File: test/util.js [hunk 2]
```diff
@@ -380,7 +380,7 @@ describe('utility methods', function() {
 			function resolveMissingFormatter() {
 				util.resolveFormatter('missing-formatter');
 			}
-			resolveMissingFormatter.should.throw(TypeError, {message: 'Invalid Formatter'});
+			resolveMissingFormatter.should.throw(/There was a problem loading formatter/);
```
Comment (shinnn) [inline test/util.js]: Submitted in https://github.com/adametry/gulp-eslint/commit/fc9200e986faa2282145c8182c1aa142961bb7ec as your commit. Thanks.

File: util.js [hunk 3]
```diff
@@ -240,11 +240,6 @@ exports.resolveFormatter = function(formatter) {
 		formatter =	CLIEngine.getFormatter(formatter) || formatter;
 	}
 
-	if (typeof formatter !== 'function') {
```
Comment (shinnn) [inline util.js]: Submitted in https://github.com/adametry/gulp-eslint/commit/fc9200e986faa2282145c8182c1aa142961bb7ec as your commit. Thanks.

**Comment by TrySound** [comment]
Thanks.

**Pull request closed without merging**